ai-data/generative-apis/how-to/use-function-calling.mdx

```diff
@@ -0,0 +1,331 @@
+---
+meta:
```

Comment:
I'm not sure this document should be a how to, since how to are usually about using core product features via the Scaleway console. Would it be more suited to being a tutorial, API/CLI doc or reference content? @bene2k1 
https://www.scaleway.com/en/docs/guidelines/#page-types

Reply:
Agreed, but I think we can make an exception for this specific product.